### PR TITLE
[UPDATE] change read.me 's project info

### LINKED ISSUE #1: [Feature]: Open-Source skills development
### Description

Learn how to work on an open-source project :rocket:

### Expected behaviour

Here is what you have to do :

- [ ] Create a project from [PoC's project template](https://github.com/PoCInnovation/open-source-project-template) and play with :
  - [ ] Milestones
  - [ ] Issues
  - [ ] Pull Requests
  - [ ] Releases
  - [ ] Labels
  - [ ] GitHub project (beta version)

- [ ] Update the [README.md](https://github.com/PoCInnovation/DagViz/blob/main/README.md) by replacing `[PROJECT'S NAME]` by `DagViz`, by filling `Our PoC Team`, ...

- [ ] Update the [CONTRIBUTING.md](https://github.com/PoCInnovation/SecureEnv/blob/main/CONTRIBUTING.md) file by replacing `[PROJECT'S NAME]` and `$REPOSITORY` with `DagViz`, ...

### Environment

/

### Proposed solution

/

### PR DESCRIPTION
**Pull request by Doozers**
# Description

Please provide a detailed description of what was done in this PR.
Precise the issue that you are resolving.

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to an issue
- [x] I have linked this PR to a milestone
- [x] I have linked this PR to a project
- [ ] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [x] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ If you're interested in how the project is organized at a higher level, please c
 ## Our PoC team :heart:
 
 Developers
-| [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy)
+| [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Ismaël FALL]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy)
```
Comment (TomChv) [inline README.md]: What about other members of the team?

**Review by TomChv — COMMENTED** [review]
Nice, I left 1 comment for one missing information

File: README.md (hunk 1 above)
Comment (adrienfort) [inline README.md]: Moreover you can remove the `[` and `]` around the developers's name
Update the links and the pictures too

Here is an [example](https://github.com/PoCInnovation/InterPlanetaryCloud#our-poc-team-ok_hand)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 > If you see this section, you've just created a repository using [PoC Innovation's Open-Source project template](https://github.com/PoCInnovation/open-source-project-template). Check the [getting started guide](./.github/getting-started.md).
```
Comment (adrienfort) [inline README.md]: You can remove this
```suggestion
```

**Review by adrienfort — CHANGES_REQUESTED** [review]
Some changes need to be done :+1: Check my comments

Also please **do not put your GitHub username in the name pour the branch** : you should have called it something like `fix/documents`.

Moreover, your PR must resolve an issue. This issue is #1 . Please link this PR to this issue.
As you can see, this issue mention the `CONTRIBUTING.md`. This PR has to fix it too.

File: README.md (hunk 1 above)
Comment (adrienfort) [reply README.md]: I think you've fixed it just before I submitted my review

Comment (Doozers) [reply README.md]: I didn't know but Florian did it

File: README.md (hunk 2 above)
Comment (Doozers) [reply README.md]: done :+1:

**Comment by adrienfort** [comment]
@Doozers and @EdenComp can you follow the commits norm explained in the [CONTRIBUTING.md](https://github.com/PoCInnovation/DagViz/blob/main/CONTRIBUTING.md) ?

**Comment by Doozers** [comment]
> Some changes need to be done +1 Check my comments
> 
> Also please **do not put your GitHub username in the name pour the branch** : you should have called it something like `fix/documents`.
> 
> Moreover, your PR must resolve an issue. This issue is #1 . Please link this PR to this issue. As you can see, this issue mention the `CONTRIBUTING.md`. This PR has to fix it too.

Yes, didn't find the section on the right but it's fix now

**Comment by adrienfort** [comment]
@Doozers @EdenComp Before merging this PR, have you played on your own with the PoC's project template ?
As mention in the issue #1

**Review by adrienfort — APPROVED** [review]
LGTM

**Pull request merged**